### DISCUSSION TITLE
Looping Music Discs

### DISCUSSION THREAD

**Discussion by hirata-estate** [PurpurMC/Purpur#641] [__Dismissed__] [answered]
Add an option for music discs to loop/play again when they end instead of having them just stop after finishing.

loop-jukeboxes: true/false

possibly an option for below for a delay between each loop.

loop-jukeboxes-delay: integer

**Comment by encode42** [comment, marked as answer]
Unfortunately, this is not possible. The server does not know when a record is finished, and clients can change records with resource packs, resulting in jank.

This is the same reason why parrots don't stop dancing after a record finishes!
